Dockerfile

```diff
@@ -714,6 +715,33 @@ RUN --mount=type=cache,target=/ccache,from=public.ecr.aws/supabase/postgres:ccac
 # Create debian package
 RUN checkinstall -D --install=no --fstrans=no --backup=no --pakdir=/tmp --nodoc
 
+####################
+# internal/supautils.yml
+####################
+FROM ccache as supautils-source
+ARG supautils_release
+ARG supautils_release_checksum
+ADD --checksum=${supautils_release_checksum} \
+    "https://github.com/supabase/supautils/archive/refs/tags/v${supautils_release}.tar.gz" /tmp/supautils.tar.gz
+RUN tar -xvf /tmp/supautils.tar.gz -C /tmp && \
+    rm -rf /tmp/supautils.tar.gz
+# Install build dependencies
+RUN apt-get update && apt-get install -y \
+    libicu-dev \
+    && rm -rf /var/lib/apt/lists/*
+# Build from source
+WORKDIR /tmp/supautils-${supautils_release}
+RUN --mount=type=cache,target=/ccache,from=public.ecr.aws/supabase/postgres:ccache \
+    make -j$(nproc)
+# Create debian package
+RUN checkinstall -D --install=no --fstrans=no --backup=no --pakdir=/tmp --nodoc
```

Comment:
Do we still need to build it from source?

Reply:
It's mostly for documentation at this point since the production stage doesn't depend on supautils-source stage. But I can also remove it to align with graphql extension